src/main/java/com/cinema/Main.java

```diff
@@ -17,12 +17,11 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
-import org.apache.log4j.Logger;
+import lombok.extern.log4j.Log4j;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
+@Log4j
 public class Main {
```

Comment:
make a collection of requests in postman to fill your database and remove the main class